<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>osagoCalcForm</class>
 <widget class="QDialog" name="osagoCalcForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Расчет осаго</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Тип транспортного средства</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="autoTypeComboBox">
         <property name="modelColumn">
          <number>1</number>
         </property>
         <property name="tableName" stdset="0">
          <string>auto_types</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="clientTypeLabel">
         <property name="text">
          <string>Собственник</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="clientTypeComboBox">
         <property name="modelColumn">
          <number>1</number>
         </property>
         <property name="tableName" stdset="0">
          <string>client_type</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Место регистрации</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="regTypeComboBox">
         <property name="modelColumn">
          <number>1</number>
         </property>
         <property name="tableName" stdset="0">
          <string>reg_type</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Регион, город</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="townLineEdit">
         <property name="tableName" stdset="0">
          <string>towns</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="classLabel">
         <property name="text">
          <string>Класс</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="classComboBox">
         <property name="modelColumn">
          <number>1</number>
         </property>
         <property name="tableName" stdset="0">
          <string>classes</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="agesLabel">
         <property name="text">
          <string>Лица, допущенные к управлению ТС</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="agesComboBox">
         <property name="modelColumn">
          <number>1</number>
         </property>
         <property name="tableName" stdset="0">
          <string>ages</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="powerLabel">
         <property name="text">
          <string>Мощность двигателя</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="powerComboBox">
         <property name="modelColumn">
          <number>1</number>
         </property>
         <property name="tableName" stdset="0">
          <string>power</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="foreignLabel">
         <property name="text">
          <string>Срок регистрации</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QComboBox" name="foreignComboBox">
         <property name="modelColumn">
          <number>1</number>
         </property>
         <property name="tableName" stdset="0">
          <string>foreign_period</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="internalLabel">
         <property name="text">
          <string>Срок регистрации</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QComboBox" name="internalComboBox">
         <property name="modelColumn">
          <number>1</number>
         </property>
         <property name="tableName" stdset="0">
          <string>internal_period</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>63</width>
           <height>27</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Размер страховой премии</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="2" rowspan="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>48</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0" rowspan="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>48</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="bonus">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>63</width>
           <height>64</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="nextButton">
         <property name="text">
          <string>Далее</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
